frontend/packages/ceph-storage-plugin/src/components/kms-config/utils.tsx

```diff
@@ -291,6 +294,7 @@ const createClusterVaultResources = (
       vaultConfigData = {
         ...vaultConfigData,
         VAULT_AUTH_KUBERNETES_ROLE: kms.authValue.value,
+        VAULT_AUTH_MOUNT_PATH: kms.providerAuthPath,
```

Comment:
From 4.10, Rook also wants Authentication Path with key name in configMap i.e VAULT_AUTH_MOUNT_PATH